conductr_cli/conductr_backup.py

```diff
@@ -61,12 +61,18 @@ def backup(args):
             backup_agents(args, backup_directory)
 
             compress_backup(args.output_path, backup_directory)
-
     finally:
         remove_backup_directory(backup_directory)
+    return True
 
 
 def compress_backup(output_path, backup_directory):
+    log = logging.getLogger(__name__)
+
+    if sys.stdout.isatty() and output_path is None:
+        log.error('Backup: Refusing to write to terminal. Provide -o or redirect elsewhere')
```

Comment:
Tiny nitpick: can you change "Backup" to "conduct backup" as the message should be the name of the binary/command